svg/painting/parsing/fill-computed.svg

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml"
+     width="800px" height="800px">
+  <title>SVG Geometry Properties: getComputedValue().fill</title>
+  <metadata>
+    <h:link rel="help" href="https://svgwg.org/svg2-draft/painting.html#FillProperty"/>
+    <h:meta name="assert" content="fill computed value is as specified, with url values absolute."/>
+  </metadata>
+  <g id="target"></g>
+  <h:script src="/resources/testharness.js"/>
+  <h:script src="/resources/testharnessreport.js"/>
+  <h:script src="/css/support/computed-testcommon.js"/>
+  <script><![CDATA[
+
+test_computed_value("fill", "none");
+test_computed_value("fill", "rgb(12, 34, 56)");
+
+test_computed_value("fill", 'url("https://example.com/")');
+test_computed_value("fill", 'url("https://example.com/") none');
+test_computed_value("fill", 'url("https://example.com/") rgb(12, 34, 56)');
+
+// context-fill and context-stroke are not yet supported by browsers.
+
+test(() => {
+  const target = document.getElementById('target');
+  target.style['fill'] = 'url("a.b#c")';
+  const result = getComputedStyle(target)['fill'];
+  assert_true(result.includes('/a.b#c'), result + ' is absolute');
```

Comment:
Is there a way to test this more precisely?

Reply:
Also: there _should_ be a test of bare hash urls (`fill="url(#x)"`). 

Advance warning: that's not currently consistently handled by browsers.

Reply:
>let resolved = new URL("a.b#c", document.URL).href;
>assert_equals('url("' + resolved + '")', result);

or so ought to work. (Or use HTMLAnchorElement.href, but URL is probably supported widely enough by now.)

Reply:
Changed to use URL